src/Views/General.vala

```diff
@@ -154,9 +161,14 @@ public class MouseTouchpad.GeneralView : Gtk.Grid {
             attach (primary_paste_help, 1, 5, 3);
 
             var xsettings = new GLib.Settings ("org.gnome.settings-daemon.plugins.xsettings");
-            primary_paste_switch.notify["active"].connect (() => on_primary_paste_switch_changed (primary_paste_switch, xsettings));
+            primary_paste_switch.notify["active"].connect (() => on_primary_paste_switch_changed (
```

Comment:
This is a pretty weird way to do this line break. It probably would make more sense to just do:
```
primary_paste_switch.notify["active"].connect (() =>  {
    on_primary_paste_switch_changed (primary_paste_switch, xsettings);
});
	
```